RandomError1.py

```diff
@@ -0,0 +1,226 @@
+from __future__ import division
+__author__ = 'PaulComeau'
+import random
+import argparse
+from sys import argv
+from argparse import ArgumentParser, ArgumentDefaultsHelpFormatter, FileType
+
+import doctest
+
+newstring=""
+FastQ=open('ABL1_E_r_001_116_R_0_GSP2_R2.fastq', 'r')
+errorchance=0
+placemark=0
+Pscorelist=[]
+
+def phredvalues_version8(line):
+    Pscore=0
+    for ASCII in line:
+        if ASCII is '"':
+            Pscore+=.79433
+        if ASCII is '#':
+            Pscore+=.63096
+        if ASCII is '$':
+            Pscore+=.50119
+        if ASCII is '%':
+            Pscore+=.39811
+        if ASCII is '&':
+            Pscore+=.31623
+        if ASCII is "'":
+            Pscore+=.25119
+        if ASCII is '(':
+            Pscore+=.19953
+        if ASCII is ')':
+            Pscore+=.15849
+        if ASCII is '*':
+            Pscore+=.12589
+        if ASCII is '+':
+            Pscore+=.10000
+        if ASCII is ',':
+            Pscore+=.07043
+        if ASCII is '-':
+            Pscore+=.06310
+        if ASCII is '.':
+            Pscore+=.05012
+        if ASCII is '/':
+            Pscore+=.03981
+        if ASCII is '0':
+            Pscore+=.03162
+        if ASCII is '1':
+            Pscore+=.02512
+        if ASCII is '2':
+            Pscore+=.01995
+        if ASCII is '3':
+            Pscore+=.01585
+        if ASCII is '4':
+            Pscore+=.01259
+        if ASCII is '5':
+            Pscore+=.01000
+        if ASCII is '6':
+            Pscore+=.00794
+        if ASCII is '7':
+            Pscore+=.00631
+        if ASCII is '8':
+            Pscore+=.00501
+        if ASCII is '9':
+            Pscore+=.00398
+        if ASCII is ':':
+            Pscore+=.00316
+        if ASCII is ';':
+            Pscore+=.00251
+        if ASCII is '<':
+            Pscore+=.00200
+        if ASCII is '=':
+            Pscore+=.00158
+        if ASCII is '>':
+            Pscore+=.00126
+        if ASCII is '?':
+            Pscore+=.00100
+        if ASCII is '@':
+            Pscore+=.00079
+        if ASCII is 'A':
+            Pscore+=.00063
+        if ASCII is 'B':
+            Pscore+=.00050
+        if ASCII is 'C':
+            Pscore+=.00040
+        if ASCII is 'D':
+            Pscore+=.00032
+        if ASCII is 'E':
+            Pscore+=.00025
+        if ASCII is 'F':
+            Pscore+=.00020
+        if ASCII is 'G':
+            Pscore+=.00016
+        if ASCII is 'H':
+            Pscore+=.00013
+        if ASCII is 'I':
+            Pscore+=.00010
+        if ASCII is 'J':
+            Pscore+=.00008
+    Pscorelist.append(Pscore)
+    return Pscorelist
+
+def phredscore(fastq_fh):
```

Comment:
Missing method level comments. See https://github.com/archerdx/analysis/blob/dev/fusions/generate_annotated_contig_bed.py for examples of how to comment methods.